src/provider/CentralServerProvider.tsx

```diff
@@ -845,6 +845,14 @@ export default class CentralServerProvider {
     return result.data as BillingOperationResponse;
   }
 
+  public async deletePaymentMethod(userID: string, paymentMethodID: string): Promise<any> {
+    const url = `${this.buildRestServerURL()}/${ServerRoute.REST_BILLING_PAYMENT_METHOD}`
+      .replace(':userID', userID)
+      .replace(':paymentMethodID', paymentMethodID);
```

Comment:
Use the new method that has been put in place:
<img width="1122" alt="2021-06-07_17-56-36" src="https://user-images.githubusercontent.com/25266945/121051766-06fcb000-c7ba-11eb-8dd4-3e28a4b32984.png">

Reply:
I'd like to do a separate PR to do the change everywhere needed ?

Reply:
> I'd like to do a separate PR to do the change everywhere needed ?

Okay